app/views/criteria/index.html.erb

```diff
@@ -9,16 +9,18 @@
     @assignment.parent_assignment.short_identifier + ' ' + PeerReview.model_name.human :
     @assignment.short_identifier) %>
 
+<% marks_released = flash[:notice].present? %>
```

Comment:
As a note related to my above comment: It's good to define this variable here, since it's a top-level view.

However it's not ideal to rely on whether the flash is set or not, since there could be other messages we have. Instead, access `@assignment` directly to compute the boolean value.